hack/boilerplate/license_header.raw.txt

```diff
@@ -1,4 +1,4 @@
-Copyright YEAR Antrea Authors
+Copyright 2020 Antrea Authors
```

Comment:
i don't remember having this changed .. by any chance your make codegen failed?

Reply:
No idea why we have this change. But "2020" should be correct?
"make codegen" did not fail.

Reply:
I think it's supposed to be YEAR here. This is the template used to inject the license into source files.

Reply:
If it is set to 2020, I believe it may work now, but it won't work any more when we switch to 2021

Reply:
yes in my case this is not cleaned up only in the event of a codegen failure.. so a little surprised this is left unchanged after codegen

Reply:
Humm..
I reran "make codegen" and got no change. Do you guys have an idea what is wrong there?